### PR TITLE
Remove feature junit.extension

### PR DESCRIPTION
**Pull request by apodhrad**
The plugin junit.extension is installed within the feature eclipse. Moreover, the plugin junit.extension depends on the plugina eclipse, workbench, etc...

### CODE REVIEW

File: features/org.eclipse.reddeer.eclipse.feature/feature.xml [hunk 1]
```diff
@@ -112,12 +119,5 @@
          install-size="0"
          version="0.0.0"
          unpack="false"/>
-   
-   <plugin
```
Comment (rawagner) [inline features/org.eclipse.reddeer.eclipse.feature/feature.xml]: why did you remove reddeer.go ?

**Comment by apodhrad** [comment]
I've got back the reddeer.go plugin
